memfault-ecs-cli/meta.yaml

```diff
@@ -0,0 +1,23 @@
+{% set name = "ecs-cli" %}
+{% set version = "1.21.0.memfault0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
```

Comment:
```suggestion
{% set version = "1.21.0" %}

package:
  name: {{ name }}
  version: {{ version }}.memfault0
```

Reply:
I went ahead and (blindly) applied this, but for my edification what's the material reason to prefer this approach?